index.html

```diff
@@ -172,6 +185,30 @@ <h3 id="active">Active</h3>
           <p>For <dfn>active fingerprinting</dfn>, we also consider techniques where a site runs JavaScript or other code on the local client to observe additional characteristics about the browser, user, device or other context.</p>
           <p>Techniques for active fingerprinting might include accessing the window size, enumerating fonts or plug-ins, evaluating performance characteristics, reading from device sensors, and rendering graphical patterns. Key to this distinction is that <a>active fingerprinting</a> takes place in a way that is potentially detectable on the client.</p>
         </section>
+        <section>
+          <h3 id="ephemeral">Ephemeral</h3>
+          <p><dfn class="lint-ignore">Ephemeral fingerprinting</dfn> is a technique to associate separate
```

Comment:
This lint was about an unused and un-exported definition, right? It's probably worth exporting it.

```suggestion
          <p><dfn class="export">Ephemeral fingerprinting</dfn> is a technique to associate separate
```